sdk/Makefile

```diff
@@ -57,7 +60,8 @@ DATA_GENERATORS = idash2017-logistic-regression-generator \
 	              string-generator \
 	              linear-regression-generator \
 	   	          number-stream-generator \
-	   	          image-processing-generator
+	   	          image-processing-generator \
+				  shamir-secret-sharing
```

Comment:
Strange indentation here?

Reply:
Probably from a rebase, can fix

Reply:
OH, this wasn't noticable in my text editor because it's a tabs v spaces issue (Makefiles notoriously need tabs)